x-pack/packages/ai-infra/kb-artifact-builder/src/command.ts

```diff
@@ -0,0 +1,97 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import Path from 'path';
+import { REPO_ROOT } from '@kbn/repo-info';
+import yargs from 'yargs';
```

Comment:
I don't think we should be updating the eslint rule, that list if the list of ignored paths within production code, that can import dev-only code. This is not what we're trying to do here.

The package is flagged as `devOnly`, and we're not importing it from any production code, aside from the `scripts` directory, so I'm surprised about those warnings (that I didn't see on my local either, tbh). 

If the cause of that warning is because we're using a dev-only package for a script in the `/scripts` directory, then I'm not sure that the recommended approach to have "dev-only" scripts would be.

Reply:
Oh no, you mean you're getting a warning in this file about some import? Then it's just the environment isolation rule being very slow to discover what is and what is not dev code. Run `kbn bootstrap` a couple time, restart your IDE, it will eventually go away.

CI would have failed if there was a real dev/prod isolation problem, those are errors not warnings.